### PR TITLE
docs: add simulation data to README

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

This adds information about what aircraft configuration is currently being simulated.

Discord username (if different from GitHub): someperson#4953

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,17 @@
 
 The A32NX Project is a community-driven open source project to create a free Airbus A320neo in Microsoft Flight Simulator that is as close to reality as possible. It aims to enhance the default A320neo by improving the systems depth and functionality to bring it up to payware-level, all for free.
 
+The following aircraft configuration is currently simulated:
+
+ ```
+ Model      A320-251N
+ Engine     LEAP 1-A26
```
Comment (wpine215) [inline README.md:17]: Should be LEAP 1A-26 iirc

**Review by NathanInnes — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -10,6 +10,17 @@
...
+The following aircraft configuration is currently simulated:
+
+ ```
+ Model      A320-251N
+ Engine     LEAP 1A-26
+ FMGS       Honeywell Pegasus Step 1A
+ FWC Std.   H2F9C
+ ```
+
+Note that those parameters might be incosistent / might change while the project evolves towards a more stable form.
```
Comment (davwheat) [inline README.md:22]:
```suggestion
Please note that this configuration may change in the future as the A32NX project evolves and changes.
```

File: README.md [hunk 3]
```diff
@@ -10,6 +10,17 @@
 
 The A32NX Project is a community-driven open source project to create a free Airbus A320neo in Microsoft Flight Simulator that is as close to reality as possible. It aims to enhance the default A320neo by improving the systems depth and functionality to bring it up to payware-level, all for free.
 
+The following aircraft configuration is currently simulated:
+
+ ```
+ Model      A320-251N
```
Comment (davwheat) [inline README.md:16]: Should we use a table for this instead?

**Review by davwheat — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Benjozork) [reply README.md:16]: Sure.

**Pull request merged**